notices.json

```
@@ -181,5 +181,23 @@
         "actionParam": "/boards/welcome"
       }
     }
-  }
+  },
+      {
+    "id": "unsupported-server-version-v5.37",
+    "conditions": {
+      "audience": "sysadmin",
+      "clientType": "all",
+      "instanceType": "onprem",
+      "serverVersion": ["<7.1"]
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Your Mattermost server version is unsupported. Upgrading is required",
+        "description": "You are on an unsupported Mattermost server version and it’s time to upgrade! [Read here](https://mattermost.com/blog/why-to-upgrade-mattermost-server/?utm_source=inapp-self-hosted&utm_medium=popup&utm_campaign=notification-why-upgrade-blog&utm_id=nov-notifications&utm_content=notification-why-upgrade-blog) on how we have improved server performance for Mattermost users, as well as why upgrading your Mattermost server ensures that you’re getting the latest new features, security fixes, and bug fixes.",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/server_upgrade.png",
+        "actionText": "Upgrade Now",
+        "actionParam": "https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html?utm_source=inapp-self-hosted&utm_medium=popup&utm_campaign=notification-upgrade&utm_id=nov-notifications&utm_content=notification-upgrade"
+      }
+    }
+  },
```

JSON does not validate. 


```suggestion
  }
```

---

@phoinixgrr I've committed the fix.